packages/gatsby-source-drupal/src/utils.js

```diff
@@ -147,10 +147,11 @@ const handleWebhookUpdate = async (
     nodesToUpdate.push(...addedReferencedNodes)
   } else {
     // if we are inserting new node, we need to update all referenced nodes
-    const newNodeReferencedNodes = referencedNodesLookup
-      .get(newNode)
-      .map(id => getNode(id))
-    nodesToUpdate.push(...newNodeReferencedNodes)
+    const newNodes = referencedNodesLookup.get(newNode)
+    if (typeof newNodes !== `undefined`) {
+      const newNodeReferencedNodes = newNodes.map(id => getNode(id))
+      nodesToUpdate.push(...newNodeReferencedNodes)
```

Comment:
Can I suggest to just push in the loop instead? Up to you.

```suggestion
      newNodes.forEach(id => nodesToUpdate.push(getNode(id)))
```